jbi/queue.py

```diff
@@ -199,13 +200,16 @@ async def get(self, bug_id: int) -> AsyncIterator[QueueItem]:
                 yield QueueItem.parse_file(path)
             except (JSONDecodeError, ValidationError) as e:
                 raise QueueItemRetrievalError(
-                    f"Unable to load item at path {path} from queue"
+                    f"Unable to load item at path {path} from queue",
```

Comment:
I think this is superflous and will come out as: `QueueItemRetrievalError: Unable to load item at path /var/lib/jbi/queue/99999/xxxxxxxxxxxxxxxxxxxxx from queue - path: /var/lib/jbi/queue/99999/xxxxxxxxxxxxxxxxxxxxx.`

Passing `path=path` should be enough